---
abstract: "The use of random forests is increasingly common in genetic association studies. The variable importance measure (VIM) that is automatically calculated as a by-product of the algorithm is often used to rank polymorphisms with respect to their ability to predict the investigated phenotype. Here, we investigate a characteristic of this methodology that may be considered as an important pitfall, namely that common variants are systematically favoured by the widely used Gini VIM. As a consequence, researchers may overlook rare variants that contribute to the missing heritability. The goal of the present article is 3-fold: (i) to assess this effect quantitatively using simulation studies for different types of random forests (classical random forests and conditional inference forests, that employ unbiased variable selection criteria) as well as for different importance measures (Gini and permutation based); (ii) to explore the trees and to compare the behaviour of random forests and the standard logistic regression model in order to understand the statistical mechanisms behind the preference for common variants; and (iii) to summarize these results and previously investigated properties of random forest VIMs in the context of genetic association studies and to make practical recommendations regarding the choice of the random forest and variable importance type. All our analyses can be reproduced using R code available from the companion website: http://www.ibe.med.uni-muenchen.de/organisation/mitarbeiter/020_professuren/boulesteix/ginibias/"
authors:
- AL Boulesteix
- admin
- JL Bermejo
- C Strobl
date: "2011-09-10T00:00:00Z"
doi: "https://doi.org/10.1093/bib/bbr053"
featured: false
image:
  caption:
  focal_point: ""
  preview_only: false
projects: []
publication: 'In *Briefings in Bioinformatics*.'
publication_short: ""
publication_types:
- "2"
publishDate: "2011-09-10T00:00:00Z"
slides:
summary:
tags:
- Random Forest
- SNP
- Gini importance
- variable importance
- machine learning
title: "Random forest Gini importance favours SNPs with large minor allele frequency: Impact, sources and recommendations"
url_code: "https://www.ibe.med.uni-muenchen.de/organisation/mitarbeiter/020_professuren/boulesteix/ginibias/index.html"
url_dataset: ""
url_pdf: https://doi.org/10.1093/bib/bbr053
url_poster: ""
url_project: ""
url_slides: ""
url_source: ""
url_video: ""
---
